src/index.html

```diff
@@ -15,8 +15,98 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#">
+          <img
+            src="images/logo.png"
+            alt="Moyo Company Logo"
+          />
```

Comment:
The 'alt' attribute should describe the image content. Make sure the description is meaningful and describes the image accurately.